app/views.py

```diff
@@ -186,6 +187,11 @@ def remove_from_cart(request, slug):
                     order.amount -= cart_item.get_final_price()
                     order.save()
                     cart_item.delete()
+                if (order.coupon == None):
```

Comment:
Nên sửa điều kiện thành 
`if (order.coupon is None):`

[Tham khảo](https://note.nkmk.me/en/python-none-usage/#checking-none-is-none-is-not-none)